autoemulate/emulators/gaussian_process/exact.py

```diff
@@ -454,3 +456,115 @@ def forward(self, x):
         assert isinstance(mean_x, TensorLike)
         covar_x = self.covar_module(x)
         return GaussianProcessLike(mean_x, covar_x)
+
+
+# GP registry to raise exception if duplicate created
+GP_REGISTRY = {
+    "GaussianProcess": GaussianProcess,
+    "GaussianProcessCorrelated": GaussianProcessCorrelated,
+}
+
+
+def create_gp_subclass(
+    name: str, gp_base_class: type[GaussianProcess], **fixed_kwargs
+) -> type[GaussianProcess]:
+    """
+    Create a subclass of GaussianProcess with given fixed_kwargs.
+
+    This function creates a subclass of GaussianProcess where certain parameters
```

Comment:
I think we should be careful with the language here - this excludes the param from the AutoEmulate tuning (in that you set a kernel) but there is still the GPyTorch tuning of the kernel hyperparameters that happens. Reading this would make me think that the GPyTorch tuning is being turned off.